garden-backend-service/src/api/routes/modal/modal_apps.py

```diff
@@ -51,19 +51,21 @@ async def add_modal_app(
     prefixed_app_name = f"{user.identity_id}-{modal_app.app_name}"
 
     # TODO: set a timeout for this and/or make it async
-    deploy_modal_app(
-        modal_app.file_contents,
-        prefixed_app_name,
-        settings.MODAL_TOKEN_ID,
-        settings.MODAL_TOKEN_SECRET,
-        settings.MODAL_ENV,
-    )
+    #  deploy_modal_app(
+    #  modal_app.file_contents,
+    #  prefixed_app_name,
+    #  settings.MODAL_TOKEN_ID,
+    #  settings.MODAL_TOKEN_SECRET,
+    #  settings.MODAL_ENV,
+    #  )
```

Comment:
We'll need to uncomment this

Reply:
Whoops! Left the comments in by accident. I wanted to avoid accidentally publishing a bunch of apps when testing manually.